modules/programs/rofi.nix

```diff
@@ -271,7 +271,9 @@ in {
     } + (optionalString (themeName != null) (toRasi { "@theme" = themeName; }));
 
     xdg.dataFile = mkIf (themePath != null) (if themePath == "custom" then {
-      "rofi/themes/${themeName}.rasi".text = toRasi cfg.theme;
+      "rofi/themes/${themeName}.rasi".text =
+        toRasi (filterAttrs (n: _: hasPrefix "@" n) cfg.theme)
```

Comment:
Isn't it preferred to have an `@import` here instead :thinking: ?

Reply:
What do you mean? This just makes sure that `@import`/`@theme` (whichever the user chooses) are sorted before other sections.

Reply:
What I meant if isn't better to be explicit about `@import`/`@theme` instead of a generic `@`. But well, you just answered that.

I think it is better to be explicitly on the tests though.

Reply:
How about this?

Reply:
Yeah there can be `@media` sections as well which start with `@` but do not need to be brought to the top of the theme file. The change seems reasonable and looks good to me now